Automattic/jetpack
# Forms: Adjust Forms inbox view columns responsiveness

## Proposed changes:
* Adjusts Forms inbox view columns responsiveness
  - From: `width: 45%` - `min-width: 160px`  
  - Date: `width: 27.5%` - `min-width: 100px`  
  - Source: `width: 27.5%` - `min-width: 100px`  
 

### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

Enable the jetpack forms package and the new dashboard before testing with:

```
add_filter( 'jetpack_contact_form_use_package', '__return_true' );
add_filter( 'jetpack_forms_dashboard_enable', '__return_true' );
```

- Go to `/wp-admin/admin.php?page=jetpack-forms`
- Check if the Inbox table columns width matches the values describe above
